doc/command.md

```diff
@@ -46,4 +46,34 @@ class MyTestCase extends WebTestCase {
 }
 ```
 
+If you need to mock a service in your command, you can use the `setServiceMock` method.
```

Comment:
Does it work only for commands? Otherwise this text may be added here: https://github.com/liip/LiipFunctionalTestBundle/blob/master/doc/basic.md

Reply:
I'm not sure. It'll take some time to work on it.

Reply:
I've added more tests and documentation, about the non-obvious nuances of the setMockService method usage.